### PR TITLE
Add NTP HOWTO section

### LINKED ISSUE #199: systemd-timesyncd documentation
Elemental runs `systemd-timesyncd` and uses the pre-configured SUSE ntp servers:
~~~
# timedatectl show-timesync --all
LinkNTPServers=
SystemNTPServers=
FallbackNTPServers=0.suse.pool.ntp.org 1.suse.pool.ntp.org 2.suse.pool.ntp.org 3.suse.pool.ntp.org
ServerName=2.suse.pool.ntp.org
~~~

This might not be wanted in all scenarios.
Esp. if time servers should be set by DHCP.

This `cloud-config` snippet is a possible workaround

~~~
write_files:
- content: |
    #! /usr/bin/bash

    [ -n "$CONNECTION_UUID" ] || exit

    INTERFACE=$1
    ACTION=$2

    case $ACTION in
        up | dhcp4-change | dhcp6-change)
            [ -n "$DHCP4_NTP_SERVERS" ] || exit
            mkdir -p /etc/systemd/timesyncd.conf.d/
            cat<<EOF > /etc/systemd/timesyncd.conf.d/$CONNECTION_UUID.conf
    [Time]
    NTP=$DHCP4_NTP_SERVERS
    RootDistanceMaxSec=15
    EOF
            systemctl restart systemd-timesyncd
            ;;
        down)
            rm -f /etc/systemd/timesyncd.conf.d/$CONNECTION_UUID.conf
            systemctl restart systemd-timesyncd
            ;;
    esac
path: /etc/NetworkManager/dispatcher.d/10-update-timesyncd
permissions: '0500'
~~~

To do
* research this topic, is this script sufficiently generic to be put into Elemental ?
* write a 'How to' section on this topic

### PR DESCRIPTION
**Pull request by fgiudici**
Fixes #199

### CODE REVIEW

File: docs/ntp.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      device: /dev/vda
+      reboot: true
+machineInventoryLabels:
+  element: fire
+
+```
+
+### Configure NTP from DHCP
+In order to get the NTP server from the network via the NTP DHCP option, we need
+a NetworkManager dispatcher script to reconfigure dinamically the `systemd-timesync` service when
```
Comment (frelon) [inline docs/ntp.md:59]:
```suggestion
a NetworkManager dispatcher script to reconfigure dynamically the `systemd-timesync` service when
```

File: versioned_docs/version-1.4/ntp.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      device: /dev/vda
+      reboot: true
+machineInventoryLabels:
+  element: fire
+
+```
+
+### Configure NTP from DHCP
+In order to get the NTP server from the network via the NTP DHCP option, we need
+a NetworkManager dispatcher script to reconfigure dinamically the `systemd-timesync` service when
```
Comment (frelon) [inline versioned_docs/version-1.4/ntp.md:59]:
```suggestion
a NetworkManager dispatcher script to reconfigure dynamically the `systemd-timesync` service when
```

File: docs/ntp.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
+---
+sidebar_label: Configure NTP
+title: ''
```
Comment (frelon) [inline docs/ntp.md:3]: Is this supposed to be empty?

**Review by frelon — APPROVED** [review]
A few small suggestions. Looks very good!

File: docs/ntp.md (hunk 3 above)
Comment (fgiudici) [reply docs/ntp.md:3]: yep, we have it empty for all the other sections...  my guess is that it is never used 🤔

**Pull request merged**